package.json

```diff
@@ -6,14 +6,13 @@
   "scripts": {
     "format": "prettier . --write",
     "check:format": "prettier . --check",
-    "check:types": "dotenv -e .env.local -- turbo run check --filter=!@u22n/web-app --cache-dir=.turbo",
+    "check:types": "dotenv -e .env.local -- turbo run check --filter=!@u22n/web --cache-dir=.turbo",
```

Comment:
remove the all the `--filter` entirely, they were there to prevent nuxt app form being checked